### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by learntomake**
Add B827EBFFFE93380F

### CODE REVIEW

**Comment by learntomake** [comment]
Added  B827EBFFFE93380F

**Comment by learntomake** [comment]
B827EBFFFE93380F.json added?

File: B827EBFFFE93380F.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE93380F", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-eu",
```
Comment (gonzalocasas) [inline B827EBFFFE93380F.json]: This is not the correct address. If you want to connect to the European router, then it should be `router.eu.thethings.network`. The full list is here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

Comment (learntomake) [reply B827EBFFFE93380F.json]: Thank you very much!

**Pull request closed without merging**